### PR TITLE
Cleanup "publishing" for NPM package

### PR DESCRIPTION
**Pull request by clepore**
### Explain the **details** for making this change. What existing problem does the pull request solve?

- [x] Remove old documentation
- [x] Change the banner name for versioning the components
- [x] Reorg the npm scripts to make more "readable" and cooperate more with CI
- [x] Rename `publish/` to its package name (aka `ids-enterprise`)

### **Steps necessary to review your pull request (required)**:

- Run all build scripts
  - [ ] `npm start`
  - [ ] `npm run publish-setup`
- [ ] Verify 0 errors and that the app works
- [ ] Check/grep documentation to make sure that I updated it to match these changes

### CODE REVIEW

**Comment by tmcconechy** [comment]
Added Artem to make sure he doesn't use the deprecated docker scripts at all to be sure.

File: docs/PUBLISH.md [hunk 1]
```diff
@@ -65,14 +65,14 @@ git checkout master
 * Change the v-Next build http://bamboo.infor.com/chain/admin/config/editChainDetails.action?buildKey=SOHO-NEXT
```
[the remark this answers is not in the document]
Comment (clepore) [reply docs/PUBLISH.md]: That whole section?

File: package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "private": true,
   "scripts": {
-    "build": "npx grunt",
     "start": "npm run build  && node app/server.js",
-    "serve": "node app/server.js",
+    "quickstart": "npx grunt demo && node app/server.js",
```
Comment (EdwardCoyle) [inline package.json]: :+1: thanks for this!

**Comment by clepore** [comment]
- [ ] Verify with artem about the `stop.sh` script change before merging

File: scripts/configs/copy.js [hunk 3]
```diff
@@ -73,16 +73,16 @@ module.exports = {
...
         },
-        { expand: true, flatten: true, src: ['src/components/locale/cultures/*.*'], dest: 'publish/dist/js/cultures/', filter: 'isFile' },
-        { expand: true, flatten: true, src: ['dist/svg/*.*'], dest: 'publish/dist/svg/', filter: 'isFile' },
-        { expand: true, flatten: true, src: ['dist/css/*theme*.*'], dest: 'publish/dist/css/', filter: 'isFile' },
-        { expand: true, flatten: true, src: ['src/**/*.scss'], dest: 'publish/sass/', filter: 'isFile' },
+        { expand: true, flatten: true, src: ['src/components/locale/cultures/*.*'], dest: 'ids-enterprise/dist/js/cultures/', filter: 'isFile' },
+        { expand: true, flatten: true, src: ['dist/svg/*.*'], dest: 'ids-enterprise/dist/svg/', filter: 'isFile' },
+        { expand: true, flatten: true, src: ['dist/css/*theme*.*'], dest: 'ids-enterprise/dist/css/', filter: 'isFile' },
+        { expand: true, flatten: true, src: ['src/**/*.scss'], dest: 'ids-enterprise/sass/', filter: 'isFile' },
         { expand: true, flatten: true, src: ['LICENSE'], dest: 'publish', filter: 'isFile' }
```
Comment (EdwardCoyle) [inline scripts/configs/copy.js]: @clepore this is causing a `publish/` folder to still show up in the project root.

**Comment by clepore** [comment]
The `stop` cmd/script seems to a bit iffy, so we can leave it for now

**Comment by tmcconechy** [comment]
Yes anything mentioning bamboo in publish.md but I'm seeing a couple old git links and stuff. 

These are old steps so we can rewrite it to be "here's what you do/need to know about releasing and publishing a version" which was the original purpose of publish.md

I can push some updates tomorrow

**Comment by clepore** [comment]
Tested manually, passed :check:

**Pull request merged**